Sources/Squirrel/ContentView.swift

```diff
@@ -96,13 +96,20 @@ struct ContentView: View {
             /// Show a header when permissions aren't granted yet.
             if !viewModel.permissionsGranted {
                 VStack(alignment: .leading, spacing: 0) {
-                    Text("Accessibility Permissions Needed")
-                        .foregroundColor(.white)
-                        .font(.title3)
-                        .fontWeight(.semibold)
-                        .frame(maxWidth: .infinity, alignment: .leading)
-                        .padding(12)
-                        .background(Color.blue.brightness(-0.2))
+                    VStack(alignment: .leading, spacing: 6) {
+                        Text("Accessibility Permissions Needed")
+                            .foregroundColor(.white)
+                            .font(.title3)
+                            .fontWeight(.semibold)
+
+                        Text("Tap to open Accessibility settings")
+                            .foregroundColor(.white.opacity(0.7))
+                            .font(.body)
+                            .fontWeight(.medium)
+                    }
+                    .padding(12)
+                    .background(Color.blue.brightness(-0.2))
+                    .frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
This should probably be reversed:

```swift
.frame(maxWidth: .infinity, alignment: .leading)
.background(Color.blue.brightness(-0.2))
```

Currently looks like this (seems to be slightly short):

<img width="300" alt="Right side of blue background is clipped" src="https://user-images.githubusercontent.com/49819455/216754364-aa386d1a-47c4-4254-ae08-ff420fcb7cef.png">

Future note: the text color contrast could be improved

Reply:
The contrast is fine. The background for the steps isn't showing up in the screenshots very well.